air/src/air/transition/mod.rs

```diff
@@ -271,15 +278,12 @@ impl<E: FieldElement> TransitionConstraintGroup<E> {
     /// them by the divisor later on. The degree of the divisor for transition constraints is
     /// always $n - 1$. Thus, once we divide out the divisor, the evaluations will represent a
     /// polynomial of degree $D$.
-    pub fn merge_evaluations<B, F>(&self, evaluations: &[F], x: B) -> E
+    pub fn merge_evaluations<B, F>(&self, evaluations: &[F], xp: B) -> E
```

Comment:
The comment above probably should be updated.

Reply:
Done, let me know what you think.